src/main/java/org/jabref/gui/entryeditor/citationrelationtab/CitationRelationsTab.java

```diff
@@ -270,6 +293,37 @@ private void styleFetchedListView(CheckListView<CitationRelationItem> listView)
         listView.setSelectionModel(new NoSelectionModel<>());
     }
 
+    private EntryEditor createEntryEditor() {
+        Supplier<LibraryTab> tabSupplier = () -> this.libraryTab;
+        return new EntryEditor(this.libraryTab,
+                // Actions are recreated here since this avoids passing more parameters and the amount of additional memory consumption is neglegtable.
```

Comment:
I think, the main reason is that undo and redo are independent here. Thus, these should be fake ones? -- The BibTeX is from the external service and NOT contained in the current library. Thus, it has NOTHING to do with the current library and thus no undo/redo relation with it.

**Update**: I think, this is not necessary at all. No entry editor is needed.